packages/web-react/src/components/ModalComposed/README.md

```diff
@@ -110,12 +110,13 @@ The rest of the properties are created from the default `<div>` element. [Docs][
 
 **Available props**
 
-| Name               | Type                        | Default   | Required | Description                   |
-| ------------------ | --------------------------- | --------- | -------- | ----------------------------- |
-| `children`         | `ReactNode`                 | -         | no       | Children node                 |
-| `align`            | `'left', 'center', 'right'` | `'right'` | no       | Children node                 |
-| `UNSAFE_className` | `string`                    | -         | no       | ModalFooter custom class name |
-| `UNSAFE_style`     | `CSSProperties`             | -         | no       | ModalFooter custom style      |
+| Name               | Type                                     | Default   | Required | Description                                                                  |
+| ------------------ | ---------------------------------------- | --------- | -------- | ---------------------------------------------------------------------------- |
+| `children`         | `ReactNode`                              | -         | no       | Children node                                                                |
+| `align`            | [Align dictionary][dictionary-alignment] | `'right'` | no       | [**DEPRECATED**][deprecated] in favor of `alignmentX`; ModalFooter alignment |
```

Comment:
```suggestion
| `align`            | [AlignmentX dictionary][dictionary-alignment] | `'right'` | no       | [**DEPRECATED**][deprecated] in favor of `alignmentX`; ModalFooter alignment |
```

Let's be concrete here. And update this everywhere. (just fulltext replace `[Align dictionary]` -> `[AlignmentX dictionary]`)

Reply:
fixed